azure-pipelines.yml

```diff
@@ -200,9 +200,19 @@ steps:
     displayName: 'Deploy to cloudroll'
 
   # TODO - delete after release pipeline creation
```

Comment:
I think ? this comment belongs with the "Publish Npm Package" task that starts on line 213. I would think that even after we split into build & release pipelines, we will need to execute this step, right?

Reply:
I moved it up here because I thought we decided to move the whole dev releasing part to the release pipeline part, should this task actually be here after the split?

Reply:
I think we need the prep for the version to be done in the build pipeline so that the artifact that will be uploaded for the release pipeline will have the correct version. When we move to build + release pipeline, the build pipeline will be responsible for producing the files (as artifacts) that will be published to npm. Does that make sense?

Reply:
Yes I think so! I'll go ahead and move the comment.